doc/DataPlaneCodeGeneration/Handle-Ambiguity.md

```diff
@@ -0,0 +1,225 @@
+# Handle Ambiguity Between Protocol Method and Convenience Method
+
+## Background
```

Comment:
instead of `Background`, i think this is more of a `Problem` corresponding to the `Solution` we have below.